src/style.css

```diff
@@ -0,0 +1,119 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.searchBar {
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 20px 0 0 ;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchBarSmall {
+  position: relative;
+  width: 100%;
+  box-sizing: border-box;
+  margin: 20px 0 0;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.searchSymbol {
+  position: absolute;
+  box-sizing: border-box;
+  width: 19px;
+  height: 19px;
+  top: 25px;
+  left: 26px;
+  z-index: 10;
```

Comment:
do not use z index instead add image as a background image

Reply:
done. thanks